tasks/task1403_check_validity_date_mmddyyyy.json

```diff
@@ -0,0 +1,1217 @@
+{
+    "Contributors": [
+        "Ravsehaj Singh Puri"
+    ], 
+    "Source": [
+        "synthetic"
+    ], 
+    "Categories": [
+        "Answer Generation"
+    ], 
+    "Definition": "In this task, you are given a date in \"mm/dd/yyyy\" format. You need to check if the date is valid or not. Return 1 if it is valid, else return 0. A date is valid is the components month(\"mm\"), day(\"dd\") and year(\"yyyy\") are all valid individually. A day(dd) is valid if it is greater than or equal to 1 and less than 30 or 31 depending upon the month(mm). Months which have 31 days are January, March, May, July, August, October, December. Rest of the months have 30 days except February which has 28 days if it is not a leap year and 29 days if it is a leap year. A month(mm) is valid if it lies in the range from 1 to 12 as there are 12 months in a year. A year is always valid if it is expressed in the form of \"yyyy\".", 
+    "Positive Examples": [
+        {
+            "input": "14/25/1405", 
+            "output": "0",
+            "explanation": "It is an invalid date as the month(mm) is 14 which does not lie in the range 1 to 12."
+        }, 
+        {
+            "input": "7/29/1617", 
```

Comment:
Here, for (mm) or (dd) if they're single-digit numbers, shouldn't they appear as, say, 07 instead of 7?